### ISSUE TITLE
Link to Control.Lens.Cons from Data.Sequence.Lens

### ISSUE THREAD

**Issue by treeowl** [ekmett/lens#780]
I suspect most people who look into `Data.Sequence.Lens` will also want to know that `Seq` is an instance of `Cons` and of `Snoc`, and that they can therefore use various facilities in `Control.Lens.Cons`. I know re-exports are widely frowned upon, but I think it would pay to at least include a note in the documentation. Perhaps something like

> `Seq` is an instance of `Cons` and of `Snoc`. This means you can use the prisms `_Cons` and `_Snoc` with sequences, along with the other functions in `Control.Lens.Cons`.

Unfortunately, my Haddock-fu is not very strong, so I don't know the right place/way to add that documentation. But I feel fairly strongly that it should go right at the top of the module.

**Comment by RyanGlScott** [comment]
I promise Haddock-fu isn't _that_ hard to learn. Just put what you've typed out [here](https://github.com/ekmett/lens/blob/989a15670bc4888491164248d6c0de389e518572/src/Data/Sequence/Lens.hs#L11)! :)

**Comment by RyanGlScott** [comment]
@wisn, sure, that's the basic idea. Care to submit a pull request?

**Comment by wisn** [comment]
@RyanGlScott Yes, I'd. However, give me some moments to playing around with Haddock.
